### PR TITLE
repo-sync-2024-06-03T15:30:33+0800

### PR DESCRIPTION
**Pull request by da-niao-dan**

### CODE REVIEW

**Review by 6fj — DISMISSED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,62 +14,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 `Fixed` for any bug fixes.
 `Security` in case of vulnerabilities.
 
-## staging
->
-> please add your unreleased change here.
+## [v1.6.1b0] - 2024-05-24
```
Comment (warriorpaw) [inline CHANGELOG.md:17]: This should be 1.7.0dev2024xxxx

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,62 +14,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-### Added
-
-- Support defenses in AutoAttack benchmark
-- SLModel: Add FSHA attack
-- SLModel: Add FedPass defense
-- Component: Add union_comp
-- Infra: Support Linux Arm build
-
-### Changed
-
```
Comment (warriorpaw) [inline CHANGELOG.md:30]: Why were these changes deleted?

Comment (da-niao-dan) [reply CHANGELOG.md:30]: rephrased to "Add arm64 build support"

**Review by 6fj — APPROVED** [review]
lgtm

**Review by warriorpaw — DISMISSED** [review]
LGTM

**Review by zhouaihui — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,62 +14,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 `Fixed` for any bug fixes.
 `Security` in case of vulnerabilities.
 
-## staging
```
Comment (zhouaihui) [inline CHANGELOG.md:17]: 和本PR没关系，v.1.6的记录是上一次就忘写了吗?

**Review by zhouaihui — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,62 +14,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-### Added
-
-- Add FSHA attack
-- Support duplicate keys in psi comp
-- Add sl/fl tf backend support custom loss
-
-### Fixed
-
-- Resolve gpu docker install error
-- Fix woe bin export err
```
Comment (ian-huu) [inline CHANGELOG.md:52]: 这里删太多了，把 1.6 的拷过来，再加上 1.7 的吧 https://github.com/secretflow/secretflow/blob/release/1.6.x/CHANGELOG.md

**Review by ian-huu — APPROVED** [review]
LGTM

**Review by 6fj — APPROVED** [review]
lgtm
